2021/45xxx/GSD-2021-45046.json

```diff
@@ -1135,1254 +1201,5 @@
             "lastModifiedDate": "2022-07-25T18:18Z",
             "publishedDate": "2021-12-14T19:15Z"
         }
-    },
-    "overlay": {
```

Comment:
Was there any data in the overlay, or was just leftover from experimentation?

Reply:
It's still in git, I have vbasicly posted correct CVSSv3/etc, removing for now.